.github/workflows/tests.yml

```
@@ -0,0 +1,38 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - ${{ github.event.repository.default_branch }}
+  pull_request:
+    branches:
+      - ${{ github.event.repository.default_branch }}
+    paths:
+      - '*.js'
+      - 'package.json'
+
+jobs:
+  tests:
+    env:
+      NPM_CONFIG_PROGRESS: "false"
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 6
```

Now that we're trying to save energy anyway, please enable [caching](https://github.com/marketplace/actions/setup-node-js-environment#caching-packages-dependencies) as well:

```suggestion
          node-version: 6
          cache: 'npm'
```